ansible_collections/arista/avd/roles/eos_cli_config_gen/templates/eos/dot1x.j2

```diff
@@ -9,6 +9,9 @@
 {%     if dot1x.system_auth_control is arista.avd.defined(true) %}
 dot1x system-auth-control
 {%     endif %}
+{%     if dot1x.protocol_bpdu_bypass is arista.avd.defined(true) %}
+dot1x protocol bpdu bypass
+{%     endif %}
 {%     if dot1x.protocol_lldp_bypass is arista.avd.defined(true) %}
 dot1x protocol lldp bypass
 {%     endif %}
```

Comment:
Please swap the order of bpdu and lldp. That is how it is rendered on EOS. Thanks.